samples/snippets/src/main/java/com/example/firestore/beam/ExampleFirestoreBeamWrite.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.firestore.beam;
+
+import com.google.cloud.firestore.FirestoreOptions;
+import com.google.firestore.v1.Document;
+import com.google.firestore.v1.Value;
+import com.google.firestore.v1.Write;
+import org.apache.beam.runners.dataflow.options.DataflowPipelineOptions;
+import org.apache.beam.sdk.Pipeline;
+import org.apache.beam.sdk.io.gcp.firestore.FirestoreIO;
+import org.apache.beam.sdk.io.gcp.firestore.RpcQosOptions;
+import org.apache.beam.sdk.options.PipelineOptions;
+import org.apache.beam.sdk.options.PipelineOptionsFactory;
+import org.apache.beam.sdk.transforms.Create;
+
+public class ExampleFirestoreBeamWrite {
+  private static final FirestoreOptions FIRESTORE_OPTIONS = FirestoreOptions.getDefaultInstance();
+
+  public static void main(String[] args) {
+    PipelineOptions options =
+        PipelineOptionsFactory.fromArgs(args).withValidation().as(PipelineOptions.class);
+    Pipeline pipeline = Pipeline.create(options);
+
+    String collectionId = "cities-collection";
+    RpcQosOptions rpcQosOptions =
+        RpcQosOptions.newBuilder()
+            .withHintMaxNumWorkers(options.as(DataflowPipelineOptions.class).getMaxNumWorkers())
+            .build();
+
+    Write write1 =
+        Write.newBuilder()
+            .setUpdate(
+                Document.newBuilder()
+                    // resolves to
+                    // projects/<your-project-id>/databases/<your-database-id>/documents/cities/NYC
```

Comment:
can we refer to the `collectionId` as `collectionId` or `databaseId` in both the variable and the comment, instead of using different terms?